reactfrontend/src/components/assetDisplay/assetDisplay.js

```diff
@@ -150,14 +150,17 @@ class AssetDisplay extends Component {
 
 		return (
 		<div className="table_container">
+			{/* is the {asset} necessary? does it show or will show sth in the future? */}
```

Comment:
Needs clarification

Reply:
what it does is it loads another component when you click on an asset and that component shows more asset credentials In the future, that's where we will be able to update those credentials

Reply:
I do not agree to resolve issues without prior engineering consultation

Reply:
Do we really need to have code that will be used in the future?

Reply:
Yeah, it's not something we need now. So the story is that I started developing this functionality and somewhere in the middle of it I have realized there is too much stuff that we are missing to actually finish it. Once we finish adding stuff like asset credentials (comments, history, etc.) only then it will make sense to come back and finish this piece of code.